dmd-script

```diff
@@ -412,11 +415,15 @@ while ( $arg_i < scalar(@ARGV) ) {
         $output_parents = 1;
     } elsif ( $arg =~ m/^-pipe$/ ) {
         push @out, '-pipe';
+    } elsif ( $arg =~ m/^-preview=(.*)$/ ) {
+	push @out, "-fpreview=$1";
```

Comment:
There are some preview/revert/transition flag names that have mixed upper and lower case, whereas the gdc equivalent is all lower case.

Reply:
Thanks for pointing that out. Currently only `-preview` has identifiers with different case but I added the lowercasing to all 3 switches.